Player/Player3D/Doll3D.gd

```diff
@@ -319,7 +319,10 @@ func setBreastsScale(breastsScale: float):
 	setBoneScaleAndOffset("DeformBreasts", breastsScale, Vector3(0.18713, 0.199727, 0.0)*mul)
 
 func setPregnancy(progress: float):
-	setBoneOffset("DeformBelly", Vector3(-0.03244, 0.706324, 0.0)*progress)
+	if(GM.pc.hasPerk(Perk.FertilityBroodmother)):
```

Comment:
This will deform bellies for every pregnant character if the player has the perk x3

instead of this check, you gotta add a function argument like `hasBigBelly: bool = false` and then you pass a true into this function from inside the BaseCharacter's softDollUpdate function if that character has the perk